### ISSUE TITLE
[🆕]: Carriera - Junior

### ISSUE THREAD

**Issue by Cadienvan** [Il-Libro-Open-Source/book#71]
Questa issue viene aperta per poter esprimere il proprio interesse verso l'argomento nel titolo.  

Chiunque sia interessato a questo argomento può mettere un 👍 su questa issue.  

Chiunque tra gli Ambassador sia interessato ad assegnarsi l'argomento per una prima versione può indicarlo in un commento o, più semplicemente, assegnare a se stesso la issue tramite l'interfaccia di GitHub.  

Chiunque sia interessato a contribuire può aggiungere un commento a questa issue.  

Chiunque sia interessato a proporre un argomento alternativo può farlo nella apposita [discussione](https://github.com/Il-Libro-Open-Source/book/discussions/3).  

N.B. Non necessariamente l'argomento sarà trattato in un capitolo a sé stante. Potrebbe essere trattato in un capitolo più ampio o in un capitolo dedicato ad un argomento correlato. Questa issue viene aperta esclusivamente per valutare l'interesse nell'argomento.

**Comment by BrianAtzori** [comment]
Ciao ragazzi! Per questo capitolo qualcuno aveva già qualche idea per i contenuti o per la struttura? Volevate dare più un'impronta del tipo "how-to" oppure roadmap da cui partire?

**Comment by AngeloAvv** [comment]
@BrianAtzori potremmo fare un mix di cose, @storrisi tu hai in mente qualcosa?

Avevo pensato innanzitutto a dare una definizione di junior (ovviamente non la classica legata al tempo), raccontando quindi di oneri e onori e inserire magari una mini FAQ senza andare troppo per punti per  evitare la stagnazione e progredire verso il ruolo di mid.

**Comment by BrianAtzori** [comment]
Mi sono assegnato il capitolo per supportare la stesura, essendo uno junior appena uscito dal bootcamp di Start2Impact conosco "bene" la posizione 🥲

Avendo comunque qualche annetto di esperienza alle spalle e avendo avuto modo di approfondire l'argomento credo di potervi dare un feedback sui contenuti avendo ancora l'occhio della "matricola", sono d'accordo con te @AngeloAvv il bias che ho visto in molti miei "colleghi" di bootcamp è proprio "sono arrivato alla fine, sono uno sviluppatore ora" quindi sicuramente è efficace una sezione FAQ che parli del percorso da junior a mid e così via!

Magari attendiamo le idee di @storrisi per capire se ha qualche spunto in più, altrimenti a me come forma quella proposta da te piace! Se ti va magari possiamo preparare le domande della FAQ insieme, visto che spesso i dubbi sono abbastanza comuni (e alcuni li ho ancora tutt'oggi 😂)

**Comment by Cadienvan** [comment]
@storrisi @BrianAtzori @AngeloAvv non è una pressione, è solo una info.. Qui siete andati avanti in privato a discuterne? Altrimenti credo varrebbe la pena che uno dei 3 mettesse giù una Table Of Contents e da lì si potrebbe commentare e procedere, che ne dite?

**Comment by AngeloAvv** [comment]
@Cadienvan con @BrianAtzori ne abbiamo parlato in passato quando ci siamo allineati sul capitolo mobile però non so se @BrianAtzori aveva già iniziato a scrivere qualcosa

**Comment by BrianAtzori** [comment]
Yep! Io e @AngeloAvv ci siamo sentiti in call per questo capitolo e per quello sul mobile tempo fa, l'anno scorso si potrebbe dire 🥲😂
Scherzi a parte eravamo rimasti che iniziavo a buttare giù una introduzione/carrellata generale per partire da una definizione di Junior per passare poi a parlare di altro delineando un path da seguire per uscire da questo ruolo... 

Al momento non ho scritto niente, tra feste e un altro paio di cose sono stato un po' pigliato! 
Se qualcuno ha in canna qualcosa da convidivere ben venga, in settimana o massimo settimana prossima dovrei iniziare a stendere qualcosa, magari ti scrivo in privato @AngeloAvv 🙌🏻

**Comment by Cadienvan** [comment]
No problem, sono amorevoli ping.
